barretenberg/cpp/src/barretenberg/crypto/ecdsa/c_bind.cpp

```diff
@@ -117,3 +120,119 @@ WASM_EXPORT void ecdsa__verify_signature_(uint8_t const* message_buf,
     ecdsa_signature sig = { r, s, v };
     *result = ecdsa_verify_signature<Sha256Hasher, secp256k1::fq, secp256k1::fr, secp256k1::g1>(message, pubk, sig);
 }
+
+// secp256r1 curve
+
+WASM_EXPORT void ecdsa_r_compute_public_key(uint8_t const* private_key, uint8_t* public_key_buf)
```

Comment:
Are these bb primitives that we need in wasm in order to use ecdsa r? I have no idea what I'm looking at here, who should inspect these changes?

Reply:
It's essentially parametrization of the methods we have for ECDSAR1, but over the K curve. I'll ask for review from someone that knows cpp, but should be safe